app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
```

Comment:
Consider adding error handling for cases where the command is not properly formatted or when the file operations fail. This will make the function more robust.